app/views/catalog/_show_default.html.erb

```diff
@@ -3,13 +3,15 @@
 <% if display_values.any? %>
   <div class="other-details">
     <h2><%= t('trln_argon.show.heading.metadata') %></h2>
-    <dl class="dl-invert">
+    <div>
```

Comment:
This div should be inside the if block so it surrounds each h3 and ul. I would maybe add a class to it -- maybe (class="metadata-field-container <%= field_name.parameterize %>").